paymentsheet/src/main/java/com/stripe/android/customersheet/StripeCustomerAdapter.kt

```diff
@@ -33,75 +35,87 @@ internal class StripeCustomerAdapter @Inject constructor(
     private var cachedCustomerEphemeralKey: CachedCustomerEphemeralKey? = null
 
     override suspend fun retrievePaymentMethods(): Result<List<PaymentMethod>> {
-        return getCustomerEphemeralKey().fold(
-            onSuccess = { customer ->
-                val paymentMethods = customerRepository.getPaymentMethods(
+        return withContext(Dispatchers.IO) {
+            getCustomerEphemeralKey().fold(
+                onSuccess = { customer ->
```

Comment:
Nit: Instead of a `fold`, you can just use a `map` here and everywhere else in the file).